packages/MarkdownEditor-Core.package/MarkdownToken.class/instance/locatedBetween.and..st

```diff
@@ -0,0 +1,4 @@
+initialize-release
+locatedBetween: precedingCharacter and: followingCharacter
+
+	^ self
```

Comment:
I think this is weird too. It you just want to return self then you could just leave it empty. I think a `self subclassResponsibility` would make more sense even.

Reply:
I convinced Felix that this is a idiom in Smalltalk to show "We intentionally left this method blank". Yes, we could make this a `subclassResponsibility` and move the empty method into `String` when it makes things clearer

Reply:
Tow questions:
-If this is the idiom why don’t we use it in reset?
-Isn’t it better to not provide a default implementation if that default implementation is doing nothing like it is here?

Reply:
- We propose do to so in #21 
 https://github.com/hpi-swa-teaching/MarkdownEditor/blob/7146c85492af2de634ff5f5ec1f6f305f38e429b/packages/MarkdownEditor-Core.package/MarkdownBlockTextStyler.class/instance/reset.st#L2-L4
- Actually, this method is a message in a communication way: we just inform the token about its adjacent characters. Thus, I think a simple "ok, nothing to do" is a sane default. As I said, I don't see a problem with `subclassResponsibility` either.